WireAuthentication/Sources/WireAuthenticationUI/Views/DetermineAuthMethod/SwitchBackendConfirmation.swift

```diff
@@ -82,16 +82,6 @@ public struct SwitchBackendConfirmation: View {
             )
         ]
 
-        if let countlyURL = backendConfig.endpoints.countlyURL {
```

Comment:
Since the countlyURL is currently not contained in the downloaded backend configurations, this is actually dead code.